org.jacktrip.JackTrip.yml

```diff
@@ -0,0 +1,50 @@
+app-id: org.jacktrip.JackTrip
+runtime: org.kde.Platform
+runtime-version: '5.15'
```

Comment:
```suggestion
runtime-version: '5.15-21.08'
```

Reply:
Is there any documentation about the runtime versioning?